luigi/configuration/toml_parser.py

```diff
@@ -61,6 +61,23 @@ def read(self, config_paths):
 
         return self.data
 
+    def defaults(self):
```

Comment:
solution 1: fill lacking functions to LuigiTomlParser. This is just dummy function so it's not my preferred one.